src/compilers/variable.cr

```diff
@@ -96,12 +96,19 @@ module Mint
           else
             "SHOULD NEVER HAPPEN"
           end
-        when Tuple(Ast::Node, Int32)
+        when Tuple(Ast::Node, Int32), Tuple(Ast::Node, Array(Int32) | Int32)
```

Comment:
`Tuple(Ast::Node, Array(Int32) | Int32)` is enough here since it's matching both cases.

```suggestion
        when Tuple(Ast::Node, Array(Int32) | Int32)
```